packages/lambda-xyz/src/__test__/health.test.ts

```diff
@@ -0,0 +1,14 @@
+import { LambdaHttpResponseAlb } from '@basemaps/lambda-shared';
+import * as o from 'ospec';
+import health from '../health';
+
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Thoughts on just disabling this eslint rule, seems to be ignored more than used.

Reply:
this is everywhere in the tests. Should I remove it from all tests or just the ones I've touched?

Reply:
Okay, I see some tests need it